frontend/src/components/layout/Header/Header.tsx

```diff
@@ -0,0 +1,50 @@
+// import React, { useEffect, useState } from 'react';
+
+import flower from '../../../assets/icons/ML_flower-icon.png';
+import login from '../../../assets/icons/ML_login_icon.png';
+import logout from '../../../assets/icons/ML_logout-icon.png';
+import moon from '../../../assets/icons/ML_moon-icon.png';
+import sun from '../../../assets/icons/ML_sun-icon.png';
+import my from '../../../assets/icons/ML_my-icon.png';
+import pencil from '../../../assets/icons/ML_pencil-icon.png';
+
+import { HeaderIconText } from "../../common/HeaderIconText";
+
+const Header = ({ blogName }: { blogName: string }) => {
+    // 추후 로그인 여부 확인
+    // const [isLogin, setIsLogin] = useState(false);
+
+    const checkIsLogin = () => {
+        return true;
+    }
+
+    return (
+        <header className="w-full h-16 flex bg-transparent font-PyeongChangPeace ">
+            <div className="basis-[40%]" />
+            <div className="basis-[20%] flex justify-center items-center">
+                <img className="w-8" src={flower} alt="logo" />
+                <span className="text-gray-700 font-semibold text-lg pl-1">{blogName}</span>
+            </div>
+            <div className="basis-[40%] flex justify-end">
+                {
+                    checkIsLogin() ? <div className="hidden sm:flex w-full justify-end">
+                        <HeaderIconText icon={pencil} alt="pencil" text="Write" />
```

Comment:
P3: 여기도 L#35처럼 줄바꿈 하는 것이 어떨까요? 혹시 prettier나 eslint쪽에서 자동으로 이렇게 만들어버리는걸까요?
```suggestion
                    checkIsLogin() 
                    ? 
                    <div className="hidden sm:flex w-full justify-end">
                        <HeaderIconText icon={pencil} alt="pencil" text="Write" />
```

Reply:
이렇게 하는게 가독성 측면에서 더 좋을 것 같네요! 수정하겠습니다.